### ISSUE TITLE
Is Transcrypt Dead?

### ISSUE THREAD

**Issue by garyrob** [TranscryptOrg/Transcrypt#850]
Another issue with the same name #650, was closed last week by @JennaSys. But as far as I can tell, there have been no nontrivial changes in 2 years. And there are a few outstanding, recent issues with no work done on them.

Is this project something that will be maintained moving ahead? It seems worth asking again. Last time it was asked, in #650, back in 2019, a lot happened after that. Will it again?

**Comment by JennaSys** [comment]
TL;DR - Yes, it will be maintained moving forward.

Despite his strong desire to work on the project, after many months, Jacques (the creator of Transcrypt) concluded that he was no longer able to devote any significant time to the project. He reluctantly decided to pass the Transcrypt torch, and I agreed to take on the responsibility of continuing to move it forward. As (bad) luck would have it, as soon as I started to take on the task of working on it, I too ended up getting busy to where I was unable to devote time on it until recently.

My first step was to get the repo moved to it's new home on GitHub, familiarize myself with the deployment steps, update some doc links and and get a new version deployed to PyPI without breaking anything. Now that those initial steps are done, my next task will be to work on some bug fixes and low hanging new feature additions and enhancements. I've been spending the last week doing some housekeeping by going through open issues, and categorizing/closing out old questions/discussions that didn't require further response to try and get a better handle on what needed addressing first.

There is quite a bit that needs to be done, and at some point I am certain to be reaching out for additional help/expertise in areas where I know my own knowledge is not sufficient. My goal is to continue to improve the usability of Transcrypt and work towards adding functionality, at the same time keeping in mind the philosophy and goals that Jacques had originally set for the project. 

I can not guarantee that there will be regularly scheduled updates, but I will keep Transcrypt moving forward to the best of my ability. If the time comes when I am no longer able to do so, I too will pass the torch. I use Transcrypt myself on a regular basis and I believe it deserves a place in the Python ecosystem. As such, I hope to see it continue to grow. I have opened up the GitHub discussion area for the repo and encourage open discussion and the sharing of ideas, usage tips, and helping out anyone that gets stuck. I will continue to monitor that area, as well as addressing any new issues that get submitted.

-John

**Comment by epoz** [comment]
Thanks for your efforts John, they are very much appreciated.
Here is another vote of confidence from a happy Transcrypt user. Have been using it for years now, very happily.

**Comment by garyrob** [comment]
Great to hear that, John! Thank you.

**Comment by ssorgatem** [comment]
Is this still true?
I see no commits or activity since April, which seems a bit concerning.
I'm considering using transcrypt to port an application to JS...

**Comment by JennaSys** [comment]
No not dead, it has just been dormant again unfortunately. "Best-laid plans" and all that happened last year. There will be updates in the upcoming months though.

**Comment by JennaSys** [comment]
Closing as answered
